smqtk_iqr/utils/bits.py

```diff
@@ -0,0 +1,190 @@
+import math
```

Comment:
We can probably reduce the need for this module by removing the `compute_hash_codes` function from the `compute_functions` module, a function that I don't think is utilized anymore (replaced with the `smqtk-nn-index-tool`).